lib/rataextra-backend.ts

```diff
@@ -316,6 +322,13 @@ export class RataExtraBackendStack extends NestedStack {
         httpRequestMethods: ['PUT'],
         targetName: 'alfrescoUpdateFileMetadata',
       },
+      {
+        lambda: alfrescoDeleteFile,
+        priority: 134,
```

Comment:
I'd recommend setting this to 140, so there's more space to manoeuvre if need be.